docs/source/howto/aforms.rst

```diff
@@ -0,0 +1,56 @@
+.. _aforms:
+
+How to extract AcroForm fields from a PDF using Python and PDFMiner
+********************************
+
+Before you start, make sure you have :ref:`installed pdfminer.six<install>`.
+
+The second thing you need is a PDF with AcroForms (as found in PDF files with fillable forms or multiple choices).
```

Comment:
While howto's are aimed at people that want to get things done (and thus probably have a PDF with acroforms at their hands), there might also be an audience that is following this tutorial to learn how to work with pdfminer.six. These people can benefit from an example pdf with an AcroForm. Maybe add one to the /samples directory and add a downloadable link here. 

Also, how do people know if there PDF has an AcroForm? Maybe you can include a section with method that can be used to check if the PDF has one.

Reply:
My problem is I can't share my PDF files... I'll see how to create some sinple ones or find examples online.

A simple method to check if there are forms would just be a check:
```python
if 'AcroForm' in doc.catalog:
```
So basically it's already there. I could change the code to return None if there's no AcroForm and make this code a method.

Reply:
If it's that simple, I would just add a sentence or two at the top. "To check if you're PDF has an AcroForm, you can simply run....".